CIP-0071/README.md

```diff
@@ -0,0 +1,246 @@
+---
+CIP: 71
+Title: Non-Fungible Token (NFT) Proxy Voting Standard
+Authors: Thaddeus Diamond <support@wildtangz.com>
+Comments-URI:
+Status: Proposed
+Type: Informational
+Created: 2022-10-11
+Post-History:
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines a standard mechanism for performing verifiable on-chain voting in NFT projects that do not have a governance token by using datums, plutus minting policies, and smart contracts.
+
+## Motivation
+
+This proposal is intended to provide a standard mechanism for non-fungible token (NFT) projects to perform on-chain verifiable votes using only their NFT assets. There are several proposed solutions for governance that involve using either a service provider (e.g., Summon) with native assets or the issuance of proprietary native assets.  However, there are several issues with these approaches:
+- Airdrops of governance tokens require minUTxO ADA attached, costing the NFT project ADA out of pocket
+- Fungible tokens do not have a 1:1 mechanism for tracking votes against specific assets with voting power
+- Sale of the underlying NFT is not tied to sale of the governance token, creating separate asset classes and leaving voting power potentially with those who are no longer holders of the NFT
+
+This standard provides a simple solution for this by using the underlying NFT to mint a one-time "ballot" token that can be used only for a specific voting topic. Projects that adopt this standard will be able to integrate with web viewers that track projects' governance votes and will also receive the benefits of verifiable on-chain voting without requiring issuance of a new native token.
+
+It is not intended for complex voting applications or for governance against fungible tokens that cannot be labeled individually.
```

Comment:
I'd love to see perhaps a concrete use case unfold in this section. Do you have a particular project as an example?

Reply:
Added 3.  I can name names or leave it generic just let me know which is preferred in CIPs.